### ISSUE TITLE
.drift file declared Views which expose int64 columns, are generating 'int' reader instead of 'bigint' reader

### ISSUE THREAD

**Issue by Adam-Langley** [simolus3/moor#2658]
I have a Table defined in dart:

(simplification)

```
ServerMachines on Table {

// our sql number is mapped to a dart BigInt which is converted into a custom 'NanoDateTime' class. 
// we're doing this because dart datetimes only have millisecond precision, we need tick (100ns)
 Int64Column get lastModificationTime =>
      int64().map(NanoDateTimeBigIntConverter()).nullable()(); 
  TextColumn get id => text().map(const GuidConverter())();
  @override
  Set<Column> get primaryKey => {id};
}
UploadMachines on Table {
 Int64Column get lastModificationTime =>
      int64().map(NanoDateTimeBigIntConverter()).nullable()();
  TextColumn get id => text().map(const GuidConverter())();
  @override
  Set<Column> get primaryKey => {id};
}
```

and a drift view (drift file):

```
CREATE VIEW machines_view AS
  SELECT * FROM server_machines WHERE id NOT IN (
    SELECT Id FROM upload_machines
  ) 
  UNION
  SELECT * FROM upload_machines;
```

The problematic section of the generated dart for the view looks like this:
```
   lastModificationTime: $ServerMachinesTable.$converterlastModificationTimen
          .fromSql(attachedDatabase.typeMapping.read(DriftSqlType.int,
              data['${effectivePrefix}last_modification_time'])),
```

You can see it's reading as an int instead of a bitInt - which causes syntax error

<img width="667" alt="image" src="https://github.com/simolus3/drift/assets/16373742/f352b60d-f709-4297-9483-9fddce9b33c2">
<img width="1017" alt="image" src="https://github.com/simolus3/drift/assets/16373742/72771c0c-6747-4153-8062-1a30bf13641b">

**Comment by simolus3** [comment]
Thanks for the report, I've fixed this in eb03ac5ff0390202e4a98bced19070594617336f.

**Comment by Adam-Langley** [comment]
Thanks @simolus3, fantastic turnaround as usual - just tested and the fix works.
